tests/lib/test_utils.py

```diff
@@ -33,3 +34,34 @@ def test_paragraphize_converts_newlines_to_paragraphs():
 def test_is_in_uk_returns_polygons_in_uk_bounding_box(alert_dict, lat, lon, in_uk):
     simple_polygons = [[[lat, lon]]]
     assert is_in_uk(simple_polygons) == in_uk
+
+
+@patch('lib.utils.boto3')
```

Comment:
I'm not sure what benefits we'd be getting here, so I'd rather we kept the current approach.

Reply:
I think we should use moto because it's less effort and consistent with our other apps. We're going to be adding more AWS functionality to this app soon, so ideally we should set a strong precedent.

On the other hand, I agree it's not essential for this PR. If I did this in a follow-up PR, would that be OK?

Reply:
Absolutely

Reply:
Fixed in https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/b09b015d8d40e13ad458852d8d0af359c5cdf340.